example_name: "highway"

centerline_waypoints:
  - [[-100.0, 0.0], [0.0, 0.0], [500.0, 0.0]]
  - [[-100.0, 3.6], [0.0, 3.6], [500.0, 3.6]]   

num_vehicles: 2
multi_vehicle_states:
  - [-50.0, 0.0, 0.0, 5.0]
  - [  0.0, 0.0, 0.0, 0.0]

vehicle_sizes:
  - [4.0, 2.0, 3.0]  # [length, width, wheelbase]
  - [4.0, 2.0, 3.0]

sv_predicted_decision_set:
  decision_switch_time: 1.0
  decisions:
    stage_1:
      - [{speed: 0.0, lane_id: 0}, {speed: 0.0, lane_id: 0}]
    stage_2:
      - [{speed: 0.0, lane_id: 0}, {speed: 0.0, lane_id: 0}]

sv_decisions:
  - [0.0, 0.0, 1.0]  # v_des_stage1, v_des_stage2, switch_time